mappings/net/minecraft/world/chunk/ChunkManager.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_51 net/minecraft/world/chunk/ChunkManager
+	METHOD method_1801 tick ()Z
```

Comment:
This seems to unload 100 chunks from the "Drop" Set, not sure if tick is accurate

Reply:
Now that I think about it, its upto the ChunkManager to do whatever it needs soooo you can disregard this comment

Reply:
disregarded